### PR TITLE
Renaming Catch Up Tree to Catch Up State

### PR DESCRIPTION
**Pull request by svv232**
This pr is part of the bitswap refactor

### CODE REVIEW

**Comment by svv232** [comment]
!ci-build-me

File: src/lib/transition_frontier/catchup_tree.ml [hunk 1]
```diff
@@ -1,31 +0,0 @@
-(* This is temporary until we decide to switch over fully to "super catchup".
```
Comment (mrmr1993) [inline src/lib/transition_frontier/catchup_tree.ml:1]: I think you meant do move this file instead of deleting it

**Review by mrmr1993 — CHANGES_REQUESTED** [review]

File: src/lib/transition_frontier/catchup_tree.ml (hunk 1 above)
Comment (svv232) [reply src/lib/transition_frontier/catchup_tree.ml]: yes i did

**Comment by svv232** [comment]
!ci-build-me

**Comment by svv232** [comment]
!ci-nightly-me

**Review by mrmr1993 — APPROVED** [review]

**Comment by svv232** [comment]
!ci-build-me

**Pull request merged**